shaders/src/common_getters.fs

```diff
@@ -47,6 +47,10 @@ HIGHP vec4 getUserTime() {
     return frameUniforms.userTime;
 }
 
+HIGHP float getUserTimeMod(float m) {
```

Comment:
Can you add a `/** @public-api **/`? It makes it easier to track our public APIs in the shaders.